### PR TITLE
TIQR-211: Publishing core to sonatype works (for SNAPSHOTs and RELEASEs)

### PR DESCRIPTION
**Pull request by dima-ko**

### CODE REVIEW

**Review by welsinga — APPROVED** [review]

File: data/build.gradle.kts [hunk 1]
```diff
@@ -203,10 +203,10 @@ publishing {
 }
 
     signing {
-        val signingKeyId = secureProperties.getProperty("signing.keyId")
-        val signingKey = secureProperties.getProperty("signing.key")
-        val signingPassword = secureProperties.getProperty("signing.password")
+        val signingKeyId = System.getenv("SIGNING_KEY_ID")
```
Comment (sarahachem) [inline data/build.gradle.kts:206]: so for data you keep the signing but not for core?

File: core/build.gradle.kts [hunk 2]
```diff
@@ -182,24 +182,21 @@ publishing {
...
+        }
+    }
 
     signing {
-        val signingKeyId = secureProperties.getProperty("signing.keyId")
-        val signingKey = secureProperties.getProperty("signing.key")
-        val signingPassword= secureProperties.getProperty("signing.password")
-        useInMemoryPgpKeys(signingKeyId, signingKey, signingPassword)
-        sign(publishing.publications["mavenAndroid"])
+        isRequired = false
```
Comment (sarahachem) [inline core/build.gradle.kts]: so no more signing here

File: data/build.gradle.kts (hunk 1 above)
Comment (dima-ko) [reply data/build.gradle.kts:206]: fixed

File: core/build.gradle.kts (hunk 2 above)
Comment (dima-ko) [reply core/build.gradle.kts]: fixed

**Pull request merged**